client/gutenberg/editor/edit-post/editor.js

```diff
@@ -34,13 +33,11 @@ function Editor( { settings, hasFixedToolbar, post, overridePost, onError, ...pr
 	};
 
 	return (
-		<StrictMode>
```

Comment:
This removal is not _strictly_ necessary here, but the console warnings are annoying.